resources/config/helpdesk.php

```diff
@@ -12,6 +12,15 @@
      */
     'userModelEmailColumn' => 'email',
 
+    /*
+     * List of users that will be ignored. They can do anything a typical customer can,
+     * however, no notifications from them will be received and only super users will
+     * be able to see their tickets.
+     */
+    'ignored' => [
+
+    ],
+
```

Comment:
There's no real indication of what to put in this array. Should mention that we're looking for email addresses here.

You could put a commented out email address inside the array to drive the point home.